src/error-request.ts

```diff
@@ -10,7 +10,19 @@ export async function errorRequest(state, octokit, error, options) {
   if (error.status >= 400 && !state.doNotRetry.includes(error.status)) {
     const retries =
       options.request.retries != null ? options.request.retries : state.retries;
-    const retryAfter = Math.pow((options.request.retryCount || 0) + 1, 2);
+    const base = (options.request.retryCount || 0) + 1;
+    let retryAfter;
+    switch (state.strategy) {
+      case "exponential":
+        retryAfter = Math.pow(2, base);
```

Comment:
You have base inverted with your power - You'll need to do:`Math.pow(base, exp)`.

Also, you might want to wrap this in `Math.round(Math.pow(base, exp))` just because `pow` returns a double

Reply:
Thanks! I renamed `base` variable to `retryCount` so that it's a bit less confusing. I also wrapped it in `Math.round` as suggested.

It reads as following now:
```
Math.round(Math.pow(2, retryCount))
```

As for the inversion, as per https://en.wikipedia.org/wiki/Exponential_backoff, exponential backoff is formed by taking the multiplicative factor (in this case 2) to the power of the number of observed events (in this case the number of retries that have already been performed). So I think it is correct now. Let me know if you wanted me to use different names though.